src/test/java/org/jenkinsci/plugins/matrixauth/integrations/casc/ExportTest.java

```diff
@@ -39,38 +36,11 @@ public void exportTestLegacy() throws Exception {
                     context.lookupOrFail(ProjectMatrixAuthorizationStrategy.class);
 
             CNode node = c.describe(authorizationStrategy, context);
-            assertNotNull(node);
-            Mapping mapping = node.asMapping();
-
-            List<CNode> permissions = mapping.get("permissions").asSequence();
-            assertEquals("list size", 18, permissions.size());
-
-            assertNull("no grantedPermissions", mapping.get("grantedPermissions"));
-            {
-                List<String> strings = Arrays.asList(
-                        "Credentials/Create:authenticated",
-                        "Credentials/ManageDomains:authenticated",
-                        "Credentials/View:authenticated",
-                        "Agent/Build:authenticated",
-                        "Agent/Connect:authenticated",
-                        "Agent/Create:authenticated",
-                        "Agent/Delete:authenticated",
-                        "Agent/Disconnect:authenticated",
-                        "Overall/Administer:anonymous",
-                        "Overall/Administer:authenticated",
-                        "Job/Build:authenticated",
-                        "Job/Cancel:authenticated",
-                        "Job/Configure:authenticated",
-                        "Job/Read:authenticated",
-                        "Run/Delete:authenticated",
-                        "Run/Update:authenticated",
-                        "View/Configure:authenticated",
-                        "View/Delete:authenticated");
-                for (CNode entry : permissions) {
-                    String value = entry.asScalar().getValue();
-                    assertTrue("list contains entry " + value, strings.contains(value));
-                }
-            }
+            assertEquals(
+                    toStringFromYamlFile(
+                            this,
+                            "/org/jenkinsci/plugins/matrixauth/integrations/casc/ExportTest/ExportTest-exportTestLegacy-global.yml"),
```

Comment:
the classpath entry should be relative, I would have thought `ExportTest-exportTestLegacy-global.yml` would work but this stuff can be a real pain sometimes and very hard to debug

Reply:
I wanted these files in the folder corresponding to the class, not in the folder for the package. (Kinda redundant with the file name now though 🤔 )

Reply:
`ExportTest/exportTestLegacy-global.yml` would work then I think?

Reply:
Huh. Never tried that. Javadoc indicates this could work.